src/compose/images.ts

```diff
@@ -514,73 +571,56 @@ async function removeImageIfNotNeeded(image: Image): Promise<void> {
 
 	const img = images[0];
 	try {
-		if (img.dockerImageId == null) {
```

Comment:
For `removeImageIfNotNeeded` changes, I'm really liking the reduction in if/else statements 👍🏼